### DISCUSSION TITLE
Is it possible to use react-router with Tauri

### DISCUSSION THREAD

**Discussion by OlaoluwaM** [tauri-apps/tauri#7899] [Q&A]
Currently working on React + Vite + Tauri application. I am looking to implement some multipage navigation, was wondering if `react-router` is still a good option in this context. Thanks!

**Comment by FabianLars** [comment]
> Is it possible to use react-router with Tauri

Yep. It should work the same as in any browser, no special configuration required iirc.

**Comment by kingwingfly** [comment]
I tried, and `tauri dev` works, but cannot bundle. `css` can not be correctly loaded.
